miso-web/src/main/webapp/scripts/hot_boxable.js

```diff
@@ -90,6 +90,19 @@ HotTarget.boxable = (function() {
             },
             pack: function(obj, flat, errorHandler) {
               // search field only
+            },
+            depends: ['discarded', 'distributed'],
+            update: function(obj, flat, flatProperty, value, setReadOnly, setOptions, setData) {
+              var applyChanges = function(source, autoSelect) {
```

Comment:
function doesn't seem to be used anywhere, and doesn't make sense for the search column